src/entity/Alerts.ts

```diff
@@ -44,6 +44,9 @@ export class Alerts {
   @Column({ type: 'bool', default: false })
   showStreakMilestone: boolean;
 
+  @Column({ type: 'bool', default: false })
+  showResetStreak: boolean;
```

Comment:
Maybe `showRecoverStreak` would be more explicit, what do you think?

Reply:
Good catch! Also that reset is actually a different thing that happens in our system too.